plugins/omar-wms-plugin/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
   }
   repositories {
     mavenLocal()
-    maven { url "${ossimMavenProxy}" }
+//    maven { url "${ossimMavenProxy}" }
```

Comment:
Does this project no longer use the ossimMavenProxy?

Reply:
Oops, I put that back in....commented that out since it originally didn't work.  But that was because I didnt' have OMAR_COMMON_PROPERTIES set.